addons/survey/models/survey_user.py

```diff
@@ -72,8 +72,11 @@ def _compute_scoring_success(self):
     def _compute_is_time_limit_reached(self):
         """ Checks that the user_input is not exceeding the survey's time limit. """
         for user_input in self:
-            user_input.is_time_limit_reached = user_input.survey_id.is_time_limited and fields.Datetime.now() \
-                > user_input.start_datetime + relativedelta(minutes=user_input.survey_id.time_limit)
+            if user_input.start_datetime:
```

Comment:
Could even do `if user_input.start_datetime and user_input.survey_id.is_time_limited:`

To simplify the code.

That way you could check `is_time_limit_reached` without checking `is_time_limited` (easier).